src/features/presence/__tests__/PresenceCollaborator.test.js

```diff
@@ -34,6 +34,8 @@ describe('features/presence/PresenceCollaborator', () => {
             const wrapper = getWrapper();
 
             expect(wrapper.find('.bdl-PresenceCollaborator').length).toBe(1);
+            expect(wrapper.exists('Avatar')).toBe(false);
+            expect(wrapper.exists('PresenceAvatar')).toBe(true);
```

Comment:
Probably a bit out of scope for this PR, but if we're checking for the avatar here, maybe it's also a good idea to check for the collaborator name/action & time text?

Reply:
Fair point, seems like there's a few code paths that should be added into the test file as well as probably converting this to testing-library but this can be addressed in a separate PR for the future